<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRaw</class>
 <widget class="QWidget" name="FormRaw">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,2">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Tone Generator</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="labelAudioOutputDevice">
        <property name="text">
         <string>output device name</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QSpinBox" name="boxFrequency">
          <property name="specialValueText">
           <string/>
          </property>
          <property name="suffix">
           <string>Hz</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>20000</number>
          </property>
          <property name="value">
           <number>440</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="boxWaveForm"/>
        </item>
        <item>
         <widget class="QPushButton" name="buttonGenerate">
          <property name="text">
           <string>Generate</string>
          </property>
          <property name="icon">
           <iconset resource="analog_signature_analyzer.qrc">
            <normaloff>:/icons/play_tone.ico</normaloff>:/icons/play_tone.ico</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Capture</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="labelAudioInputDevice">
        <property name="text">
         <string>input device name</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QGroupBox" name="groupTriggerMode">
          <property name="title">
           <string>Trigger Mode</string>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0">
            <widget class="QRadioButton" name="boxTriggerAuto">
             <property name="text">
              <string>Auto</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QRadioButton" name="boxTriggerNormal">
             <property name="text">
              <string>Normal</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QRadioButton" name="boxTriggerSingle">
             <property name="text">
              <string>Single</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QComboBox" name="boxTriggerChannel">
             <item>
              <property name="text">
               <string>track left channel</string>
              </property>
              <property name="icon">
               <iconset resource="analog_signature_analyzer.qrc">
                <normaloff>:/icons/blue_line.ico</normaloff>:/icons/blue_line.ico</iconset>
              </property>
             </item>
             <item>
              <property name="text">
               <string>track right channel</string>
              </property>
              <property name="icon">
               <iconset resource="analog_signature_analyzer.qrc">
                <normaloff>:/icons/green_line.ico</normaloff>:/icons/green_line.ico</iconset>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupTriggerSlope">
          <property name="title">
           <string>Trigger Slope</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="boxSlopeRising">
             <property name="text">
              <string>Rising</string>
             </property>
             <property name="icon">
              <iconset resource="analog_signature_analyzer.qrc">
               <normaloff>:/icons/front_slope.ico</normaloff>:/icons/front_slope.ico</iconset>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="boxSlopeFalling">
             <property name="text">
              <string>Falling</string>
             </property>
             <property name="icon">
              <iconset resource="analog_signature_analyzer.qrc">
               <normaloff>:/icons/back_slope.ico</normaloff>:/icons/back_slope.ico</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupChannels">
          <property name="title">
           <string>Channels</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QCheckBox" name="boxChannelLeft">
             <property name="text">
              <string>Left</string>
             </property>
             <property name="icon">
              <iconset resource="analog_signature_analyzer.qrc">
               <normaloff>:/icons/blue_line.ico</normaloff>:/icons/blue_line.ico</iconset>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="boxChannelRight">
             <property name="text">
              <string>Right</string>
             </property>
             <property name="icon">
              <iconset resource="analog_signature_analyzer.qrc">
               <normaloff>:/icons/green_line.ico</normaloff>:/icons/green_line.ico</iconset>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCapture">
        <property name="text">
         <string>Capture</string>
        </property>
        <property name="icon">
         <iconset resource="analog_signature_analyzer.qrc">
          <normaloff>:/icons/capture_data.ico</normaloff>:/icons/capture_data.ico</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="OscilloscopeView" name="oscilloscope" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>100</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OscilloscopeView</class>
   <extends>QWidget</extends>
   <header location="global">OscilloscopeView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="analog_signature_analyzer.qrc"/>
 </resources>
 <connections/>
</ui>
